hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/PipelineSyncTask.java

```diff
@@ -55,6 +55,7 @@ public class PipelineSyncTask extends ReconScmTask {
 
   private ReadWriteLock lock = new ReentrantReadWriteLock(true);
   private final long interval;
+  private final ReconTaskStatusCounter taskStatusCounter;
```

Comment:
this is already present in base class, do not need as another member, but can be access via base method to access and call further method.